docs/content/fsdocs-default.css

```diff
@@ -540,6 +543,8 @@ body {
 /* comment */
 #fsdocs-content span.c {
     color: #808080;
+    font-weight: 400;
+    font-style: italic;
```

Comment:
@gbaydin Any recollection what this was changing? thanks

Reply:
That's about making the code comments a bit lightweight and italic. Somewhat influenced by tensorflow docs style.